tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/AspNetCoreDistTraceRequestChainFixture.cs

```diff
@@ -106,9 +106,11 @@ private void WriteApplicationAgentLogToTestLogger(string applicationName, Remote
         public override void ShutdownRemoteApplication()
         {
             FirstCallApplication.Shutdown();
+            FirstCallApplication.WaitForExit();
```

Comment:
We were trying to get data from the log immediately after signaling shutdown, but since the process is set to send data on exit, sometimes we would miss it